src/js/blocks/hero/block.js

```diff
@@ -0,0 +1,379 @@
+									<span class="cl-button">
+									<PlainText
+										onChange={ content => setAttributes({ button: content }) }
+										value={ attributes.button }
+										placeholder={ __( 'Your button text' ) }
+										keepPlaceholderOnFocus={true}
+									/></span>
+									{ meta }
+								</div>
+							</div>
+						</div>
+					</div>
+				</div>
+			);
+		};
+
+		// Generate block controls for alignment, etc
+		const createBlockControls = () => {
+			return (
+				<BlockControls key="controls">
+					<BlockAlignmentToolbar
+						value={ attributes.alignment }
+						onChange={ content => setAttributes({ alignment: content }) }
+					/>
+
+					{ !! attributes.img && (
+					<MediaUploadCheck>
+						<Toolbar>
+							<MediaUpload
+								onSelect={ media =>
+									{
+										setAttributes({
+											alt: media.alt,
+											img: media.url,
+											mediaID: media.id
+										});
+									}
+								}
+								allowedTypes={ ALLOWED_MEDIA_TYPES }
+								value={ attributes.mediaID }
+								render={ ( { open } ) => (
+									<IconButton
+										className="components-toolbar__control"
+										label={ __( 'Edit media' ) }
+										icon="edit"
+										onClick={ open }
+									/>
+								) }
+							/>
+						</Toolbar>
+					</MediaUploadCheck>
+					) }
+				</BlockControls>
+			);
+		};
+
+		// Generate sidebar inspector controls for other custom attributes
+		const createInspectorControls = () => {
+			return (
+				<InspectorControls>
+					<PanelBody>
+						<PanelRow>
+							<BaseControl
+								label={ __( 'Format' ) }
+							>
+								<ButtonGroup aria-label={ __( 'Hero Format' ) }>
+									{ [ 'default', 'fullwidth', 'super' ].map( ( value ) => {
+										const capitalizedValue = value.charAt( 0 ).toUpperCase() + value.slice( 1 );
+										const key = ( 'default' === value ) ? '' : value;
+										const format = ( undefined == attributes.format ) ? '' : attributes.format;
+										const isSelected = ( key === format );
+
+										return (
+											<Button
+												key={ key }
+												isDefault
+												isPrimary={ isSelected }
+												aria-pressed={ isSelected }
+												onClick={ content => setAttributes({ format: key }) }
+											>
+												{ capitalizedValue }
+											</Button>
+										);
+									} ) }
+								</ButtonGroup>
+							</BaseControl>
+						</PanelRow>
+
+						<PanelRow>
+							<TextControl
+								label="Video URL"
+								onChange={ content => setAttributes( { vid: content } ) }
+								value={ attributes.vid }
+								className="meta-field vid"
+								help="For creating a video hero."
+							/>
+						</PanelRow>
+
+						<PanelRow>
+							<ToggleControl
+								label="Use WordPress caption"
+								help="Setting a custom caption below will override any WordPress caption."
+								checked={ attributes.use_caption }
+								onChange={ content => setAttributes( { use_caption: content } ) }
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Identifier 'use_caption' is not in camel case. (camelcase)](https://app.codacy.com/manual/uriweb/uri-component-library/pullRequest?prid=4757242)